templates/deploy-ansible-service-broker.template.yaml

```diff
@@ -101,7 +120,7 @@ objects:
     selector:
       app: ansible-service-broker
     strategy:
-      type: Recreate
+      type: Rolling
```

Comment:
I think we want to leave this as recreate.  See https://github.com/openshift/ansible-service-broker/pull/511

Reply:
Rolling will be better now that we don't have to worry about the PV that etcd is using.

Reply:
Just some additional notes:

- I tested redeploying the broker and etcd :+1: 
- I tested scaling up the broker :+1:
- I tested scaling up etcd :-1:, this is because the replicated pod thinks it's a master...and it is not. I do not think this is something this PR should be concerned with. Dealing with etcd replicas is a pain and the whole reason I think CoreOS came out with the idea of operators. Just wanted to point it out.

Reply:
>  this is because the replicated pod thinks it's a master...and it is not. I do not think this is something this PR should be concerned with. Dealing with etcd replicas is a pain and the whole reason I think CoreOS came out with the idea of operators. Just wanted to point it out.

Agree, I could see adding a trello card to make this work better.